### DISCUSSION TITLE
My own vercel instance doesn't work as expected

### DISCUSSION THREAD

**Discussion by Krishnacore** [anuraghazra/github-readme-stats#2234] [Q&A] [answered]
Like many people, I encountered this problem - `maximum retries exceeded`. I looked at the issues and realized that I need my own instance. I followed the instructions, everything was clear and straightforward. But after I deployed the application the link for some reason leads to your repository. And I am left with the same problem. I tried adding my custom domain, but that doesn't help. Thank you!

**Comment by rickstaa** [comment, marked as answer]
@Krishnacore Thanks for your question. We updated our deployment guide last month, so the guide found at https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance should be up to date. The link when clicking a card is not related to the server but to the markdown code used. The following code will direct users to https://github.com:

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com)
```

If you are still getting the `maximum retries exceeded` error after deployment, the most likely cause is that something went wrong while adding the `PAT` to Vercel. See steps 10 and 11 of the deployment guide.

![image](https://user-images.githubusercontent.com/17570430/198215400-7554f9b1-797f-43af-8d75-7db2aa06d72e.png)
